AppCenterAnalytics/AppCenterAnalytics/Internals/Model/MSEventLog.m

```diff
@@ -98,11 +98,49 @@ - (void)setPropertiesAndMetadataForCSLog:(MSCommonSchemaLog *)csLog {
   if (self.typedProperties) {
     csProperties = [NSMutableDictionary new];
     metadata = [NSMutableDictionary new];
+    NSString *baseDataPrefix = [NSString stringWithFormat:@"%@.", kMSDataBaseData];
```

Comment:
Synced offline. We should validate baseType and baseData and then remove properties if one of them is missing.